src/covjson_pydantic/i18n.py

```diff
@@ -9,4 +9,10 @@ class LanguageTag(str, Enum):
     undefined = "und"
 
 
-i18n = Dict[LanguageTag, str]
+# TODO: This was throwing warning:
+#   Expected `definition-ref` but got `LanguageTag` - serialized value may not be as expected
+# This may be a bug in Pydantic: https://github.com/pydantic/pydantic/issues/6467
+# or: https://github.com/pydantic/pydantic/issues/6422
+# So, for now, reverted to a less strict type
+# i18n = Dict[LanguageTag, str]
```

Comment:
Hmm, annoying. Do we want to track this with a ticket?

Reply:
Yes: https://github.com/KNMI/covjson-pydantic/issues/3